treeherder/perf/migrations/0007_merge.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
To avoid a merge migration, you should remove this and 0006_performancesignature_subtests.py and generate a new migration file with ./manage.py makemigrations